FLIR/conservator/wrappers/media.py

```diff
@@ -68,6 +68,7 @@ class MediaType(QueryableType, FileLockerType, MetadataType):
     metadata_gen_url = Mutation.generate_signed_metadata_upload_url
     metadata_confirm_url = Mutation.mark_annotation_as_uploaded
 
+    @staticmethod
     def verify_md5(self, local_path, expected_md5):
```

Comment:
I believe you drop `self` on static methods. I think there's a call that should be updated from `self.verify_md5` to `MediaType.verify_md5` to keep things consistent

Reply:
Yes you do when you're paying attention :)